app/_src/gateway/kong-enterprise/secrets-management/secrets-rotation.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Secrets Rotation
+---
+
+Secrets rotation is a process of updating secrets. Regular rotations of secrets is considered a good
+practice. Here are some reasons why secrets rotation is important:
+
+* Reducing impact of compromised secrets
+* Enhancing resilience against brute-force attacks
+* Complying with security regulations
+* Maintaining separation of duties
+* Adapting to evolving threats
+* Mitigating the effects of insider threats
+
+Kong needs to hold many types of secrets to be able to process requests, and to integrate with other
+systems and clients:
+
+* TLS certificates
+* Credentials and API keys (to access databases, identity providers, and other services)
+* Cryptographic keys (digital signing and encryption)
+
+There are two main ways to rotate secrets:
+
+* Rotate periodically using TTLs (e.g. check for new TLS certificate once per day)
+* Rotate on failure (e.g. on database authentication failure, check if the secrets were updated, and try again)
+
+Kong supports both methods for rotating secrets. Rotation on failure requires code to be written,
+and thus it has limited support in Kong (Postgres credentials for now). There is an experimental
+Kong PDK API that can be used to rotate secrets on failure: [kong.vault.try](/gateway/{{page.kong_version}}/plugin-development/pdk/kong.vault/#kongvaulttrycallback-options).
+
+## Periodically rotating secrets using TTLs
+
+Kong employs a background job that runs periodically on **every minute** (thus you cannot currently
+automatically rotate more often than that). Its job is to rotate secrets that are about to expire.
+The TTL can be configured  with the Vault (for all the secrets) or in secret reference (for a single secret).
+By default, Kong does not rotate any secrets, so **remember to configure the TTLs** if you want to turn on
+**the automatic rotation**.
+
+The TTL based rotation works with most of the Kong supported vaults, including:
+
+* [AWS Secrets Manager](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/backends/aws-sm/)
+* [GCP Secrets Manager](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/backends/gcp-sm/)
+* [HashiCorp Vault](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/backends/hashicorp-vault/)
+
```

Comment:
What follows seems to be a description of how the rotation process works.  I think this paragraph would better be located in the beginning of the chapter.